cpp/log/cert_test.cc

```diff
@@ -449,6 +484,13 @@ TEST_F(CertTest, SignatureAlgorithmMatches) {
   EXPECT_FALSE(mismatched_algs.IsSignedBy(issuer).ValueOrDie());
 }
 
+#if defined(OPENSSL_IS_BORINGSSL) && defined(BORINGSSL_201603)
```

Comment:
How about making this test always run, but EXPECT_FALSE with that particular BoringSSL and EXPECT_TRUE with all others versions of OpenSSL/BoringSSL?